front-end/src/view/components/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import { ArrowRight } from 'phosphor-react'
+
+interface ButtonProps {
+    name: string
+}
+
+export default function Button({ name }: ButtonProps) {
+    return (
+        <>
+            <button type="submit" className="w-full sm:w-auto flex items-center justify-center gap-2 bg-primary-color hover:bg-primary-color-houver transition-colors duration-150 ease-linear text-color-font font-inter py-3 px-12 rounded-lg cursor-pointer" >{name} <ArrowRight /></button>
```

Comment:
It would be good to also receive the type of the button as a prop, similar to the name. Also, the <ArrowRight /> will be optional, and we might have different types of icons to be used in the button. So it would be good to receive this as a property as well.